docs/Events/listening-to-events.md

```diff
@@ -9,3 +9,71 @@ parent: "Events"
 ---
 
 # Listening to Events
+
+`bpmn-js` uses a general purpose [event bus](../Modules/common-modules/event-bus.md) provided by `diagram-js` to communicate across a diagram instance based on [`events`](./bpmn-events).
+
+## Retrieving the `eventBus` Instance
+
+In order to register a listener for an `event`, the `eventBus` instance needs to be retrieved first. This instance is registered as named diagram service in an internal class called `BaseViewer` and can therefore be retrieved using the `BaseViewer#get` method which is available in the sub-classes [`NavigatedViewer`](tbd), [`Viewer`](tbd) or [`Modeler`](tbd).
+
+Alternatievely the `#on` and `#off` methods can be used in [`NavigatedViewer`](tbd), [`Viewer`](tbd) or [`Modeler`](tbd) to directly register or remove a previously registered event listener.
+
+### Example Usage
+
+Notice that the following snippet demonstrates the retrieval of the `eventBus` instance using a [`Modeler`](tbd). In principle this instance could also be retrieved from an instance of a [`NavigatedViewer`](tbd) or [`Viewer`](tbd) likewise.
+```javascript
+import BpmnJS from "bpmn-js/lib/Modeler";
```

Comment:
Let's keep it consistent and use single quotes, cf. https://bpmn-io.github.io/docs.bpmn.io/general/api-reference/bpmn-js/importXML

> import BpmnJS from 'bpmn-js/lib/Modeler';